### PR TITLE
Honour Markdown in list blocks

### PR DESCRIPTION
**Pull request by KaitCrawford**

### CODE REVIEW

**Comment by KaitCrawford** [comment]
@miltontony @saeedmb8 This is basically a copy of the fix in [bwise](https://github.com/praekelt/molo-ndoh-yep/pull/108). Will having two templatetags with the same name cause problems when bwise tries to import the files?

File: molo/core/cookiecutter/scaffold/{{cookiecutter.directory}}/{{cookiecutter.app_name}}/templates/core/article_page.html [hunk 1]
```diff
@@ -25,14 +25,22 @@ <h3 class="article-title">{{self.title}}</h3>
       {% if block.block_type == 'numbered_list' %}
         <ol>
         {% for item in block.value %}
-          <li>{{item}}</li>
+          <li>{{ item|handle_markdown }}</li>
+        {% endfor %}
+        </ol>
+      {% else %}
+      {% if block.block_type == 'list' %}
```
Comment (miltontony) [inline molo/core/cookiecutter/scaffold/{{cookiecutter.directory}}/{{cookiecutter.app_name}}/templates/core/article_page.html]: As simon pointed out on the other side, you can use `{% elif .. %}`

**Comment by miltontony** [comment]
Also made a note on the other side that you can use [this test](https://github.com/praekelt/molo/blob/develop/molo/core/tests/test_views.py#L198-L210) to test the markdown

**Comment by KaitCrawford** [comment]
@miltontony @hodgestar @saeedmb8 ready for re-review

**Comment by miltontony** [comment]
:+1: :cake:

**Pull request merged**